upgrade/v2.5/Makefile

```diff
@@ -0,0 +1,14 @@
+dist/clis:
```

Comment:
Since you have real files that this is downloading/creating it would probably be better to have had a separate target for each file (where the target is actually the filename, e.g. `dist/calicoctl-v1.4`)

But functionally this still works ok - so not fussed.